test/unit/index.html

```diff
@@ -6,6 +6,8 @@
 	<title>shoestring Test Suite</title>
 	<link rel="stylesheet" href="qunit/qunit.css" media="screen">
 	<script src="qunit/qunit.js"></script>
+	<script src="sinon/sinon-1.12.1.js"></script>
+	<script src="sinon/sinon-ie-1.12.1.js"></script>
```

Comment:
I'm slightly reluctant to introduce a big dependency even in our test suite for two tests. Hacking a little mock object for the require and pulling the call to `xmlHTTP` into a custom method might work but also might be clunky. I'll think about this.